contents/founders/write-great-job-ads.md

```diff
@@ -0,0 +1,92 @@
+featuredImageType: full
+tags:
+  - Culture
+  - People
+crosspost:
+  - Blog
+  - Newsletter      
+---
+
+At PostHog, we've written job ads that have seen 9,000 applicants in the last 12 months. One thing we've noticed in 2024 is that the quality of the average applicant has gone way, way up. We think this is due to a couple of things:
+
+- We've started putting a lot more effort into making our job ads more interesting and differentiated
+ 
+- We've stopped advertising anywhere except on our own website (of course we still pop up on a few scraper sites)
+
+Given the number of people I've interviewed recently who have said 'I loved your job ad', we figure some of this stuff must be working! Here is a super-quick tactical guide which will hopefully help. 
+
+## Obvious ways
+
+These are table stakes. If you're not doing these, it's probably not even worth looking at the non-obvious stuff. Yes I am going to mention salary again, like every other nano-influencer on LinkedIn.
+
+**Really tight requirements**
+
+Keep your list of requirements as short as possible. I like to do:
+
+- Must have
+
+- Nice to have
+
+Each should be ~3-4 items long - that's it. You will know when you see a great resume anyway. Candidates don't read this stuff super carefully anyway, or will ignore the stuff they can't do (especially if they are a dude). Fun fact - you do not need to write out literally every single thing the person will be doing. 
+
+**What the actual hiring process is**
+
+This is nice for the candidate, but more importantly it forces _you_ to figure out how you are going to decide whether a candidate is great or not. A series of random chats with the team is not going to give you the signal you need. 
+
+Even if you are a tiny team, you can still take 15min to figure this out and save yourself a lot of pain improvising the next step and getting stressed you haven't got back to a candidate. 
+
+**Time zones**
+
+All-remote doesn't necessarily mean hiring from literally anywhere. Please specify time zones before people waste a bunch of time, only to find out you don't hire in Australia. This also forces you to clarify your own thinking.
+
+**Salary**
+
+Stick a range in there if you really don't know. You may live somewhere this is legally mandated anyway. The point here is to avoid wasting a ton of your _own_ time getting someone through your process, only to find you are miles apart here. 
+
+You are not that great a negotiator, and you might as well screen out the people who reeeeally care about the money first and foremost. 
+
+## Non-obvious ways
+
+**Get the hiring manager to write it, and then a good writer to edit it**
+
+Classic process is hiring manager comes up with a list of bullets -> passes to recruiter to write the ad -> publish. This is a great way to ensure that you have a weird frankenstein ad written by two people, neither of whom are professional writers. 
```

Comment:
```suggestion
Classic process is hiring manager comes up with a list of bullets -> passes to recruiter to write the ad -> publish. This is a great way to ensure that you have a weird Frankenstein's monster of an ad written by two people, neither of whom are professional writers. 
```

Reply:
this has such 'acshully' vibes I almost rejected it lol